src/components/Counter.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Counter.css';
+
+class Counter extends React.Component {
+  state = {
+    count: 0,
```

Comment:
переменные лучше называть существительными
```suggestion
    counter: 0,
```